src/main/java/Task.java

```diff
@@ -0,0 +1,140 @@
+import java.util.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+public class Task {
+    public String type;
+    public String name;
+    public String status;
+    public String date;
+    public LocalDate processedDate;
+    public String fileDate;
+    public boolean hasProcessedDate = false;
+    public int priority = 0;
+
+    public Task(String type, String name) {
+        this.type = type;
+        this.name = name;
+        this.status = "[✗]";
```

Comment:
Nothing wrong, but I noticed that sometimes the tick and cross cannot be displayed properly (like when I run the code on my Windows 10 PC and store the output into a .txt file). Not sure if you have encounterd similar issue.